javalib/src/main/scala/java/net/StandardProtocolFamily.scala

```diff
@@ -0,0 +1,22 @@
+package java.net
+
+sealed class StandardProtocolFamily(name: String, ordinal: Int)
+    extends _Enum[StandardProtocolFamily](name, ordinal)
+    with ProtocolFamily {
+  override def toString() = this.name
+}
+
+object StandardProtocolFamily {
+  val INET = new StandardProtocolFamily("INET", 0)
+  val INET6 = new StandardProtocolFamily("INET6", 1)
+
```

Comment:
An  implementers choice suggestion.

It looks like Java 16 added a UNIX value to the enum.  Scala 
Native does not yet support Unix domain sockets (what is your
next project?) and we only advertise Java 8 support.

I suggest adding  the definition here for completeness.
Usually with a line line "// Since: Java 16".
One less change to make as we try to roll javalib support into the 
2020s. 

I think I would add UNIX